docs/platforms/android/troubleshooting/index.mdx

```diff
@@ -4,6 +4,16 @@ description: "Troubleshoot and resolve common issues with the Android SDK."
 sidebar_order: 9000
 ---
 
+## Obfuscated stack traces, missing Android-specific context
+
+If stack traces don't get unobfuscated properly, even though <PlatformLink to="/enhance-errors/proguard/">mapping files are uploaded correctly</PlatformLink>, or if Android context information is missing from events, a possible reason is that the SDK is initialized with `Sentry.init` instead of `SentryAndroid.init`.
```

Comment:
```suggestion
If stack traces don't get deobfuscated properly, even though <PlatformLink to="/enhance-errors/proguard/">mapping files are uploaded correctly</PlatformLink>, or if Android context information is missing from events, a possible reason is that the SDK is initialized with `Sentry.init` instead of `SentryAndroid.init`.
```